### ISSUE TITLE
LaTex does not seem to be visible in dark mode

### ISSUE THREAD

**Issue by hakuro-jp** [silverbulletmd/silverbullet-katex#3]
It seems that the LaTex is not visible when using dark mode.  I presume that this is simply because the font is black and dark mode now has a black background color, at least that seems to be the case.  At any rate, it is not currently useful with dark mode.

**Comment by zefhemel** [comment]
Right, yeah it’d have to detect dark mode is on and adapt the styles… 🤔

**Comment by maxthier** [comment]
Is there an update on this? It seems like it still isn't fixed.
